src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -521,34 +521,55 @@ def prepare_logs_argument(c):
             c.argument('routes_json', type=str, help="The JSON array of API routes.", validator=validate_routes)
             c.argument('routes_file', type=str, help="The file path of JSON array of API routes.", validator=validate_routes)
 
-    for scope in ['spring-cloud build-service buildpacks-binding create',
-                  'spring-cloud build-service buildpacks-binding set']:
+    # BuildpackBinding in Builder
+    for scope in ['spring-cloud build-service builder buildpack-binding create']:
         with self.argument_context(scope) as c:
             c.argument('type',
                        arg_type=get_enum_type(v20220101_preview_AppPlatformEnums.BindingType),
-                       help='Required type for buildpacks binding.')
+                       help='Required type for buildpack binding.')
             c.argument('properties',
                        help='Non-sensitive properties for launchProperties. Format "key[=value]".',
                        nargs='*',
-                       validator=validate_buildpacks_binding_properties)
+                       validator=validate_buildpack_binding_properties)
             c.argument('secrets',
                        help='Sensitive properties for launchProperties. '
                             'Once put, it will be encrypted and never return to user. '
                             'Format "key[=value]".',
                        nargs='*',
-                       validator=validate_buildpacks_binding_secrets)
+                       validator=validate_buildpack_binding_secrets)
+            c.argument('name', help='Name for buildpack binding.', validator=validate_buildpack_binding_not_exist)
+            c.argument('builder_name', help='The name for builder.', default="default")
+            c.argument('service', service_name_type, validator=only_support_enterprise)
+
 
-    for scope in ['spring-cloud build-service buildpacks-binding create']:
+    for scope in ['spring-cloud build-service builder buildpack-binding set']:
         with self.argument_context(scope) as c:
-            c.argument('name', help='Name for buildpacks binding.', validator=validate_buildpacks_binding_not_exist)
+            c.argument('type',
+                       arg_type=get_enum_type(v20220101_preview_AppPlatformEnums.BindingType),
+                       help='Required type for buildpack binding.')
+            c.argument('properties',
+                       help='Non-sensitive properties for launchProperties. Format "key[=value]".',
+                       nargs='*',
```

Comment:
doesn't aware of this, nargs cannot take care of comma, let me fix it.

Reply:
BTW, you can still use the `for scope in ['create', 'set']:` for the common parameters. After that define the different part.

```python
for scope in ['spring-cloud build-service builder buildpack-binding create', 'spring-cloud build-service builder buildpack-binding set']:
        with self.argument_context(scope) as c:
            c.argument('type',
                       arg_type=get_enum_type(v20220101_preview_AppPlatformEnums.BindingType),
                       help='Required type for buildpack binding.')
            c.argument('properties',
                       help='Non-sensitive properties for launchProperties. Format "key[=value]".',
                       nargs='*',
                       validator=validate_buildpack_binding_properties)
            c.argument('secrets',
                       help='Sensitive properties for launchProperties. '
                            'Once put, it will be encrypted and never return to user. '
                            'Format "key[=value]".',
                       nargs='*',
                       validator=validate_buildpack_binding_secrets)
            c.argument('builder_name', help='The name for builder.', default="default")
            c.argument('service', service_name_type, validator=only_support_enterprise)

with self.argument_context(.... create):
        c.argument('name', help='Name for buildpack binding.', validator=validate_buildpack_binding_not_exist)
```

Reply:
align with the env part, and keep as is.

Reply:
Both work for me. But if the for only one item can be omitted